framework_tvb/tvb/core/neocom/_h5loader.py

```diff
@@ -83,8 +83,12 @@ class DirLoader(object):
 
     def __init__(self, base_dir, registry, recursive=False):
         # type: (str, Registry, bool) -> None
+        self.base_file = None
         if not os.path.isdir(base_dir):
-            raise IOError('not a directory {}'.format(base_dir))
+            self.base_file = base_dir
+            base_dir = os.path.dirname(base_dir)
```

Comment:
This code is miss-leading. 
We name `base_dir` the parameter, and then we cut one folder up. 
I suspect that it might be a file_path given here, but the it should be the responsibility of who is calling to API o adapt.